cdap-ui/server/config/development/cdap.json

```diff
@@ -1,155 +1,13 @@
-  "dataset.executor.container.instances": "1",
-  "log.base.dir": "logs/avro",
-  "metrics.dataset.hbase.stats.report.interval": "60",
-  "http.service.boss.threads": "1",
-  "data.tx.max.instances": "5",
-  "security.token.distributed.parent.znode": "/cdap/security/auth",
-  "http.service.worker.threads": "10",
-  "security.enabled": "false",
-  "explore.start.on.demand": "false",
-  "kafka.zookeeper.namespace": "kafka",
-  "hdfs.namespace": "/cdap",
-  "data.tx.server.io.threads": "2",
-  "stream.container.instances": "1",
-  "app.worker.threads": "10",
-  "data.tx.bind.address": "127.0.0.1",
-  "metrics.max.instances": "5",
-  "metrics.connection.backlog": "20000",
-  "security.token.digest.algorithm": "HmacSHA256",
-  "data.tx.memory.mb": "512",
-  "stream.worker.threads": "10",
-  "app.exec.threads": "20",
-  "stream.bind.address": "127.0.0.1",
-  "data.tx.snapshot.dir": "/cdap/tx.snapshot",
-  "dashboard.bind.port": "11011",
-  "stream.partition.duration": "3600000",
-  "log.collection.root": "data/logs",
-  "app.boss.threads": "1",
-  "master.service.num.cores": "2",
-  "explore.inactive.operation.timeout.secs": "3600",
-  "security.server.token.expiration.ms": "86400000",
-  "security.auth.server.ssl.bind.port": "10010",
-  "data.tx.janitor.enable": "true",
-  "stream.base.dir": "/streams",
-  "data.tx.snapshot.interval": "300",
-  "router.bind.address": "127.0.0.1",
-  "data.local.inmemory.persistence.type": "LEVELDB",
-  "http.service.exec.threads": "20",
-  "kerberos.auth.enabled": "false",
-  "log.retention.duration.days": "7",
-  "hdfs.lib.dir": "/cdap/lib",
   "router.server.address": "127.0.0.1",
-  "data.tx.client.provider": "thread-local",
-  "explore.enabled": "true",
-  "metrics.num.cores": "2",
-  "metrics.query.bind.address": "127.0.0.1",
-  "dataset.service.output.dir": "/datasets",
-  "twill.no.container.timeout": "120000",
-  "metrics.worker.threads": "10",
-  "data.tx.discovery.service.name": "transaction",
-  "metrics.processor.memory.mb": "512",
-  "stream.async.worker.threads": "10",
-  "dataset.service.bind.address": "127.0.0.1",
-  "stream.event.ttl": "9223372036854775807",
-  "security.authorization.enabled": "false",
-  "dashboard.ssl.disable.cert.check": "false",
-  "data.tx.num.instances": "1",
-  "router.webapp.bind.port": "20000",
-  "router.bind.port": "11015",
-  "metrics.boss.threads": "1",
-  "metrics.data.table.retention.resolution.1.seconds": "7200",
-  "data.tx.bind.port": "15165",
-  "metrics.processor.max.instances": "5",
-  "metrics.processor.num.cores": "1",
-  "router.forward.rule": "11015:gateway,20000:webapp/$HOST",
-  "hdfs.user": "yarn",
-  "security.token.digest.keylength": "128",
-  "zookeeper.session.timeout.millis": "40000",
-  "security.token.digest.key.expiration.ms": "3600000",
-  "app.output.dir": "/programs",
-  "enable.alpha.preview": "false"
+  "dashboard.bind.port": "11011",
+  "enable.alpha.preview": "false",
+  "dashboard.router.check.timeout.secs": "0"
```

Comment:
I think it currently works even though this parameter wasn't here, because `undefined < 0` returns false, so the if-condition block doesn't execute:
https://github.com/caskdata/cdap/blob/release/3.5/cdap-ui/server/config/router-check.js#L91-L91